x/evm/keeper/grpc_query_test.go

```diff
@@ -759,3 +759,65 @@ func (suite *KeeperTestSuite) TestEstimateGas() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestTraceTx() {
+	ctx := sdk.WrapSDKContext(suite.ctx)
+	//TODO deploy contract that triggers internal transactions
```

Comment:
can you create issues for all these TODOs?

Reply:
Done!